### DISCUSSION TITLE
Issues running docker image with nvidia GPU

### DISCUSSION THREAD

**Discussion by Millertaker** [open-webui/open-webui#8155] [Issues]
I'm currently facing issues when i try to run the cmd:

```
# docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

23b09f7666adc342806f0b7ffa2a674b6a10aef1d5e77b111a2417e5a1dba14a
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy'
nvidia-container-cli: initialization error: load library failed: libnvidia-ml.so.1: cannot open shared object file: no such file or directory: unknown.
```

looks like that `libnvidia-ml.so` lib isn't compatible with the NVIDIA container tookit, or at least that's what i find out, the nvidia-docker2 looks like is deprecated so i'm unable to use the GPU for the webUI use.

**Comment by hillzzz** [comment]
I am using an Nvidia Jetson AGX Orin with Open-WebUI and it runs fine except for one CUDA issue with search. I run Ollama natively, and launch Open-Webui with this command "sudo docker run -d --gpus=all -v --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434  --name open-webui --restart always ghcr.io/open-webui/open-webui:cuda. Note the cuda at the end and  the "--gpus=all" flag. It works great so give that a try.
